Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG BASE_SERVER_IMAGE=temporalio/base-server:1.10.0
+ARG BASE_BUILDER_IMAGE=temporalio/base-builder:1.9.0
+
+##### UI builder #####
+FROM ${BASE_BUILDER_IMAGE} AS ui-builder
+
+WORKDIR /home/ui-builder
+
+RUN apk add --update --no-cache npm
+RUN npm install -g pnpm
+
+COPY package.json pnpm-lock.yaml ./
+RUN pnpm install
+
+COPY .babelrc .npmrc *.json *.yaml *.cjs *.js *.ts ./
+COPY src src
+
+ARG TEMPORAL_PUBLIC_PATH
+ARG UI_TARGET=local
```

Comment:
for consistency should we rename the arg TEMPORAL_PUBLIC_PATH to `UI_PUBLIC_PATH`?